public/app/plugins/datasource/prometheus/variables.ts

```diff
@@ -48,11 +45,4 @@ export class PrometheusVariableSupport extends StandardVariableSupport<Prometheu
 
     return metricFindStream.pipe(map((results) => ({ data: results })));
   }
-
-  toDataQuery(query: StandardVariableQuery): PromQuery {
-    return {
-      refId: 'PrometheusDatasource-VariableQuery',
-      expr: query.query,
-    };
-  }
```

Comment:
@bohandley 
Without this I am unsure how this works with old dashboards? I can only see the migration being done for the variable query editor not when queries are executed. But maybe I am missing it

Reply:
I think I know why it works still now, so don't worry about the comment above